app/assets/javascripts/oxalis/constants.js

```diff
@@ -111,10 +112,16 @@ export const POSITION_REF_REGEX = /#\(([0-9]+,[0-9]+,[0-9]+)\)/g;
 // There is an outer yellow CSS border and an inner (red/green/blue) border
 // that is a result of the plane being smaller than the renderer viewport
 export const OUTER_CSS_BORDER = 2;
-const INNER_RENDERER_BORDER = 2;
-export const ORTHOGONAL_BORDER = OUTER_CSS_BORDER + INNER_RENDERER_BORDER;
-const PLANE_WIDTH = 376;
-const VIEWPORT_WIDTH = PLANE_WIDTH + ORTHOGONAL_BORDER * 2;
+const VIEWPORT_WIDTH = 376;
+export const ensureSmallerEdge = false;
+
+// Using the following dimensions for the address space,
+// the look up buffer (256**2) is used at a rate of ~ 97%
+// ((32 × 32 × 50 + 16 × 16 × 50) / 256^2 = 0.976563)
+export const addressSpaceDimensions = {
```

Comment:
Just to understand correctly: This means from the anchor point on, at most 32 buckets in x/y direction and 50 buckets in z direction can be put into the lookup buffer?

Reply:
Yes, this is correct. If/when we convert this code to hashing at some point, these limitation would be lifted.